assets/airports/kpvd.json

```diff
@@ -0,0 +1,6536 @@
+            ]
+        }
+    ],
+    "runways": [
+        {
+            "name": ["5", "23"],
+            "end": [
+                ["N41d42m49.7723", "W071d26m05.8486", "50.8ft"],
+                ["N41d43m49.6961", "W071d25m15.5354", "44.3ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["16", "34"],
+            "end":  [
+                ["N41d43m53.9542", "W071d25m55.8304", "53.7ft"],
+                ["N41d43m06.8846", "W071d25m05.9925", "32.8ft"]
+            ],
+            "ils": [false, true]
+        }
+    ],
+    "sids": {
+        "NASA1": {
+            "icao": "NASA1",
+            "name": "Rocket Man One",
+            "body": ["#000"],
+            "rwy": {
+                "5": [],
+                "16": [],
+                "23": [],
+                "34": []
+            },
+            "exitPoints": {
+                "ROCKET": ["_MAN"]
+            },
+            "draw": [
+                ["_ONE"]
+            ]
+        }
+    },
+    "stars": {
+        "JORDN2": {
+            "icao": "JORDN2",
+            "name": "Jordan Two",
+            "entryPoints": {
+                "HTO": ["HTO"]
+            },
+            "body": ["JORDN", "^MINNK", "#015"],
+            "rwy": {
+                "5": [],
+                "16": [],
+                "23": [],
+                "34": []
+            },
+            "draw": [
+                ["HTO*", "JORDN", "MINNK"]
+            ]
+        },
+        "WIPOR3": {
+            "icao": "WIPOR3",
+            "name": "Wipor Three",
+            "entryPoints": {
+                "ALB": ["ALB", "SHIGY"],
+                "HINGZ": ["HINGZ"],
+                "HNK": ["HNK", "HINGZ"]
+            },
+            "body": ["PONEE", "NELIE", "WIPOR", "ORW", "SAUGY"],
+            "rwy": {
+                "5": ["^LAFAY", "#129"],
+                "16": ["WAWIK", "SEABE", "^NOSUH", "#047"],
+                "23": ["WAWIK", "SEABE", "^NOSUH", "#047"],
+                "34": ["^LAFAY", "#129"]
+            },
+            "draw": [
+                ["ALB*", "SHIGY", "PONEE"],
+                ["HNK*", "HINGZ*", "PONEE"],
+                ["PONEE", "NELIE", "WIPOR", "ORW", "SAUGY"],
+                ["SAUGY", "LAFAY"],
+                ["SAUGY", "WAWIK", "SEABE", "NOSUH"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "KPVD",
+            "destination": "",
+            "category": "departure",
+            "route": "BAF",
+            "method": "random",
+            "altitude": "",
+            "speed": "",
+            "rate": 2,
+            "airlines": [
+                ["aal", 7],
+                ["dal", 4],
+                ["jbu", 2],
+                ["swa", 8],
+                ["ual", 4]
+            ]
+        },
+        {
+            "origin": "KPVD",
+            "destination": "",
+            "category": "departure",
+            "route": "JUMPR",
```

Comment:
`JUMPR` is inside the airspace so this route should be extended to prevent penalisation